LuckParser/Resources/tmplBoonTable.html

```diff
@@ -14,7 +14,7 @@
 		<tr>
 			<td>{{:player.group}}</td>
 			<td title="{{:player.profession}}"><img src="{{:player.icon}}" alt="{{:player.profession}}" class="icon"><span style="display:none">{{:player.profession}}</span></td>
-			<td class="text-left"{{if !~generation}} title="Average number of boons: {{:data.avg}}"{{/if}}>{{:player.name}}</td>
+			<td class="text-center"{{if !~generation}} title="Average number of {{if ~condition}}conditions{{else}}boons{{/if}}: {{:data.avg}}"{{/if}}>{{:player.name}}</td>
```

Comment:
Hmmm I would stick with a left-aligned name column. Other tables have that too, and especially with the icon column in the left it looks cleaner when the icon is always directly next to the player name.

Reply:
The reason I ended up putting it was because the tool tips were not aligned with the text, they appear from the center while the names are on the left and it was kinda bothering me.

Reply:
Still, the layout of the table is more important than the layout of the tooltips?